framework/src/problems/ReferenceResidualProblem.C

```diff
@@ -114,6 +128,30 @@ ReferenceResidualProblem::ReferenceResidualProblem(const InputParameters & param
 
   _accept_mult = params.get<Real>("acceptable_multiplier");
   _accept_iters = params.get<int>("acceptable_iterations");
+
+  const auto norm_type_enum = params.get<MooseEnum>("normalization_type");
```

Comment:
use `params.get<MooseEnum>("normalization_type").getEnum<NormalizationType>()`  and make it an enum class

Reply:
Oh in the header and everything with a new `NormalizationType`? Is that necessary when this object only lives in the constructor?